### ISSUE TITLE
Search tokens are missing some common tags ("danish", "praying")

### ISSUE THREAD

**Issue by kenchris** [nolanlawson/emoji-picker-element#11]
I guess it is hard to handle all these, but with GBoard I can search for Danish or Denmark and it will give me the Danish flag - here only Denmark works

Also 'pray' shows praying hands but not 'praying'. Also high fives on GBoard will give you praying hands :)

**Comment by nolanlawson** [comment]
Thanks for opening the issue! It looks like this may be an issue with `emojibase` itself; I notice that their data for the Danish flag includes:

```json
{
    "annotation": "flag: Denmark",
    "shortcodes": [
      "flag_dk"
    ],
    "tags": [
      "DK",
      "flag"
    ]
  },
```

So unfortunately it contains the strings `denmark` and `dk` but not `danish`. :confused:

Same thing for :pray:, it looks like it doesn't include `praying`:

```json
{
    "annotation": "folded hands",
    "shortcodes": [
      "pray"
    ],
    "tags": [
      "ask",
      "hand",
      "please",
      "pray",
      "thanks"
    ]
}
```

Arguably I could include some English-based tokenization here, but I didn't want to include too much NLP stuff (especially if it needed to be adapted for each language). Maybe there is an extension to `emojibase` with extra tags?

**Comment by nolanlawson** [comment]
I filed an issue on the upstream library: https://github.com/milesj/emojibase/issues/75 .

Potentially, this is something that will have to be solved on our side of things, but I'm not sure yet if I want to include a full English stemmer (e.g. Porter stemmer) or demonym expansions (Danish -> Denmark, Ireland -> Irish, England -> English) in `emoji-picker-element`. That seems like a huge responsibility, especially to cover all the languages supported by `emojibase`, and it would no doubt increase the size and scope of the project. For now I'm inclined to defer to whatever `emojibase` supports.

Closing for now, thanks for opening the issue!
